libr/bin/bin.c

```diff
@@ -505,9 +503,10 @@ R_API RBinPlugin *r_bin_get_binplugin_by_name(RBin *bin, const char *name) {
 R_API RBinPlugin *r_bin_get_binplugin_by_bytes(RBin *bin, const ut8 *bytes, ut64 sz) {
 	RBinPlugin *plugin;
 	RListIter *it;
-	if (!bin || !bytes) {
-		return NULL;
-	}
+
+	r_return_val_if_fail (bin, NULL);
```

Comment:
why not use bin && bytes

Reply:
so that you know what is wrong in case an error is shown.